egs/gale_arabic/s5b/local/split_wer_per_corpus.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Report WER for reports and conversational
+# Copyright 2014 QCRI (author: Ahmed Ali)
```

Comment:
I actually have the same question, no idea where it comes from. However, calculating wer per program or per corpus may be interesting to extend it further to be per genre in cases like MGB